routers/web/shared/user/header.go

```diff
@@ -110,3 +111,21 @@ func RenderUserHeader(ctx *context.Context) {
 	defer profileClose()
 	ctx.Data["HasProfileReadme"] = profileReadmeBlob != nil
 }
+
+func LoadHeaderCount(ctx *context.Context) error {
+	prepareContextForCommonProfile(ctx)
+
+	_, repoCount, err := repo_model.SearchRepository(ctx, &repo_model.SearchRepoOptions{
```

Comment:
No, you need a count function, but not use search function here.

Reply:
Is there a Count function which checks if a User is able to see a Repo?

Reply:
Looks like there is not, but you can write one like this.
```go
func CountRepository(opts *repo_model.SearchRepoOptions) (int64,error) {
     return sess.
			Where(repo_model.SearchRepositoryCondition(opts)).
			Count(new(Repository))
}
```

Reply:
Done